doc/migration/4.x-5.0.md

```diff
@@ -10,6 +10,8 @@
 
 4. Run the following script to export the currently available StatuteTypes: [DumpStatutes.java](https://gist.github.com/jcarvalho/90205db88544e340dc2e). Keep the generated file.
 
+5. Run the following scripts to export the currently available IngressionTypes : [IngressionTypes](https://gist.github.com/NunoPinheiro/fee7caa80bbfaf757b21). Keep the generated file.
```

Comment:
This script has many of the issues described [here](https://github.com/FenixEdu/fenixedu-academic/pull/712#issuecomment-100997879). I suggest you rewrite this script based on the amended version of the other one.